src/Hl7.Fhir.Specification/Specification/Snapshot/ElementDefnMerger.cs

```diff
@@ -429,10 +430,26 @@ internal static FhirString constrainMax(FhirString max, UnsignedInt minValue)
                             onConstraint(mergedItem);
                         }
                     }
+                    if (!string.IsNullOrEmpty(source))
+                    {
+                        InitializeConstraintSource(result, source);
+                    }
+
                 }
                 return result;
             }
 
+            internal static void InitializeConstraintSource(IEnumerable<ElementDefinition.ConstraintComponent> constraints, string source)
```

Comment:
Is it true this is not needed in the R4 version?

Reply:
This is backported from R4 ;)